doc/source/whatsnew/v1.0.0.rst

```diff
@@ -169,6 +169,16 @@ You can use the alias ``"boolean"`` as well.
    s = pd.Series([True, False, None], dtype="boolean")
    s
 
+.. _whatsnew_1000.custom_window:
+
+Defining custom windows for rolling operations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We've added a :func:`pandas.api.indexers.BaseIndexer` class that allows users to define how
```

Comment:
Will need to add this somewhere in docs/source/reference I think.

Reply:
Good idea. Added in `doc/source/reference/window.rst`